LinuxTraceLogCapture.md

```diff
@@ -98,9 +98,9 @@ $ sudo lttng destroy
 ```
 
 # Perf
-Perf is used to collect CPU Sampling (cpu-clock) events as LTTng doesn't support capturing these yet. Note: Stacks may require symbol setup
+Perf is used to collect tracepoint events.
```

Comment:
Ditto - additive. Maybe "Perf is used to collect CPU Sampling (cpu-clock) events as LTTng doesn't support capturing these yet. Perf can also collect tracepoint events"